Gemfile

```
@@ -80,3 +80,4 @@ group :test do
 end
 
 gem 'carrierwave'
+gem 'i18n_generators', '~> 2.2', '>= 2.2.2'
```

```suggestion
gem 'i18n_generators'
```

特別な理由がない限りバージョンは指定しなくても良いと思います

---

i18n_generatorsのバージョンを削除しました。